packages/fxa-settings/src/components/PageTwoStepAuthentication/index.tsx

```diff
@@ -125,6 +126,7 @@ export const PageTwoStepAuthentication = (_: RouteComponentProps) => {
 
   const [verifyTotp] = useMutation(VERIFY_TOTP_MUTATION, {
     onCompleted: () => {
+      alertTextExternal('Two-step authentication enabled');
```

Comment:
This reminded me that I wanted to refactor `AlertBar` to store the state in the cache so that we wouldn't need to use external for this type of thing